plugin/pkg/scheduler/api/types.go

```diff
@@ -32,6 +32,10 @@ type Policy struct {
 	Priorities []PriorityPolicy
 	// Holds the information to communicate with the extender(s)
 	ExtenderConfigs []ExtenderConfig
+	// RequiredDuringScheduling affinity is not symmetric, but there is an implicit PreferredDuringScheduling affinity rule
+	// corresponding to every RequiredDuringScheduling affinity rule.
+	// HardPodAffinitySymmetricWeight represents the weight of implicit PreferredDuringScheduling affinity rule, in the range 0-100.
+	HardPodAffinitySymmetricWeight int
```

Comment:
Did not have requirements on `HardPodAffinitySymmetricWeight per RequiredDuringScheduling`, so prefer to keep current behaviour :). And it's complex to achieve that.